src/pages/home/report/ReportActionCompose.js

```diff
@@ -181,6 +181,11 @@ class ReportActionCompose extends React.Component {
             if (this.shouldFocusInputOnScreenFocus && this.props.isFocused) {
                 this.focus(false);
             }
+            
+            // it will focus the input when isFocused from props came true (it will be true when it is a new chat.) and will check that the device is focusable or not.
```

Comment:
The comment does not match with the code. Please correct the comment.

Reply:
I changed it but please any suggestions?